src/monitor/group_state_machine.c

```diff
@@ -403,6 +404,9 @@ ProceedGroupState(AutoFailoverNode *activeNode)
 		/* start replication */
 		AssignGoalState(activeNode, REPLICATION_STATE_CATCHINGUP, message);
 
+		/* edit synchronous_standby_names to add the new standby now */
+		AssignGoalState(primaryNode, REPLICATION_STATE_APPLY_SETTINGS, message);
```

Comment:
This is not part of the log above

Reply:
Fixed now, and moved to another place to also fix the other part of your review. Thanks!